src/Migrations/Version20231031212442.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Shipping Slot plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusShippingSlotPlugin\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20231031212442 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE monsieurbiz_shipping_slot_config CHANGE rrules rrules JSON DEFAULT NULL');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE monsieurbiz_shipping_slot_config CHANGE rrules rrules JSON DEFAULT NULL');
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏I think it depends on the version of doctrine and mysql, we could do like Sylius: https://github.com/Sylius/Sylius/blob/336682edb5be6e2885bde0179816891315d89777/src/Sylius/Bundle/CoreBundle/Migrations/Version20211018130725.php

```suggestion
        $this->addSql('ALTER TABLE monsieurbiz_shipping_slot_config CHANGE rrules rrules LONGTEXT DEFAULT NULL');
```

Reply:
@jacquesbh I did it in the PR #21